lib/cell.rb

```diff
@@ -37,16 +37,17 @@ def register_hit
         @ship.hit
     end
 
-    def render
-        if @fired_upon == false && @empty == true
-            "." # default empty cell
-        elsif @fired_upon == false && @empty == false
-            "S" # SHIP -cell containing a ship that hasn't been hit
-        elsif @fired_upon == true && @empty == true
+    def render(visibility = false) #need to play with the argument and then create
+        #another test for if vis = true vs false "." or "s" should print
+        if visibility == false && !fired_upon?
```

Comment:
This was a pain but good job to Austin for solving this and getting the tests to pass!